frontend/docs/admin-guide/getting-started.md

```diff
@@ -4,66 +4,71 @@ outline: deep
 
 # High Level Overview
 
-What is {{ $params.APPLICATION_NAME }}?
+## What is {{ $params.APPLICATION_NAME }}?
+
 - {{ $params.APPLICATION_NAME }} is an auto-provision system that enables users with valid identity provider (IdP) credentials to securely interact with Amazon AI/ML APIs through a custom AWS console-like UI without needing direct access to the AWS console.
-- {{ $params.APPLICATION_NAME }} creates secure data science environments and allows users to self organize and collaborate via projects
-- {{ $params.APPLICATION_NAME }} allows the creation of Notebooks instances (traditionally backed by AWS Sagemaker) which are machine learning optimized compute instances to conduct analysis and experimentation. For big data and compute intensive workloads these instances can also be attached to AWS EMR clusters.
+- {{ $params.APPLICATION_NAME }} creates secure data science environments and allows users to self-organize and collaborate via projects.
+- {{ $params.APPLICATION_NAME }} allows the creation of Notebook instances (traditionally backed by AWS SageMaker), which are machine learning optimized compute instances for conducting analysis and experimentation. For big data and compute-intensive workloads, these instances can also be attached to AWS EMR clusters.
+
+## What makes {{ $params.APPLICATION_NAME }} different from AWS SageMaker?
```

Comment:
Same here. We might want to avoid adding parameter in the header. maybe just change the wording?